src/Components/TodosList.js

```diff
@@ -0,0 +1,29 @@
+import TodoItem from './TodoItem';
+
+/* const TodosList = (props, setTodos) => {
+    return (
+      <ul>
+        {props.todosProps.map((todo) => (
+          <TodoItem setTodos={setTodos} itemProp={todo} />
+        ))}
+      </ul>
+    );
+  };
+  export default TodosList; */
+
+const TodosList = ({
+  todosProps, handleChange, delTodo, setUpdate,
+}) => (
+  <ul className="item">
+    {todosProps.map((todo) => (
+      <TodoItem
+        key={todo.id}
+        itemProp={todo}
+        handleChange={handleChange}
+        delTodo={delTodo}
+        setUpdate={setUpdate}
+      />
+    ))}
+  </ul>
+);
```

Comment:
### Well done but you need to check for the prop error that you are facing.. You can try some codes as shown below..

**`import PropTypes from 'prop-types';

function TodosList({todosProps, handleChange, delTodo, setUpdate}) {
  // Component implementation

TodosList.propTypes = {
  todosProps: PropTypes.array.isRequired,
  handleChange: PropTypes.func.isRequired,
  delTodo: PropTypes.func.isRequired,
  setUpdate: PropTypes.func.isRequired,
};`**